server.js

```diff
@@ -10,7 +10,7 @@ const { matematSummary, matematBuy } = require('./matemat');
 
 const SPACEAPI_URL = "http://spaceapi.hq.c3d2.de:3000/spaceapi.json";
 
-const TEST_URL_REGEX = /([-a-zA-Z0-9@:%_\+.~#?&//=]{2,256}\.[a-z]{2,4}\b(\/[-a-zA-Z0-9@:%_\+.~#?&//=]*)?)/gi;
+const TEST_URL_REGEX = /https?:\/\/[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/gi;
```

Comment:
```suggestion
const TEST_URL_REGEX = /https?:\/\/([-a-zA-Z0-9@:%_\+~#=]{1,256}\.){1,32}[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/gi;
```


This wouldn't match consecutive dots.

The first two would match but the last one wouldn't which is matched right now.

``` 
http://supersandro.de
http://a.supersandro.de
http://www...............supersandro.de
``` 

@Mic92 What do you think about this?

Reply:
Maybe :) ?

Reply:
Your group match is not quite right I think.... for `https://example.com./foo` it will match `./foo` in the group which should not be problematic because it will be substituted but maybe it will lead to some problems...

Reply:
This was also matched before. I don't want to over complicate this right now. Maybe this can be improved in another PR.